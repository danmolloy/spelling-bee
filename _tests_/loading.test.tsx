import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom"
import Loading from "../components/loading";

describe("<Loading />", () => {
  beforeEach(() => {
    render(<Loading />)
  })
  it("loading-div is in the document", () => {
    const loadingDiv = screen.getByTestId("loading-div")
    expect(loadingDiv).toBeInTheDocument()
  })
  it("matches snapshot", () => {
    const loadingDiv = screen.getByTestId("loading-div")
    expect(loadingDiv).toMatchSnapshot()
  })
})