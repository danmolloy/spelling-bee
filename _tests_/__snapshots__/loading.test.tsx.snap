// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> matches snapshot 1`] = `
<div
  class=" animate-pulse h-4 w-full bg-zinc-200 rounded-md m-1"
  data-testid="loading-div"
/>
`;
